scarlet2/source.py

```diff
@@ -31,6 +31,35 @@ def __call__(self):
     def bbox(self):
         return self.spectrum.bbox @ self.morphology.bbox
 
+class StaticSource(Module):
+    spectrum: Spectrum
+    morphology: Morphology 
+    specind: Specind
```

Comment:
What's the purpose of `Specind`? It looks to be a simple box, which you can set for the spectrum already.

Reply:
Also, it's better to use class names that are CamelCase and descriptive, so that we don't have to guess what the class does.